### PR TITLE
fix: bug: 修复交叉表手动设置滚动不生效问题

### PR DESCRIPTION
**Pull request by xingwanying**
### 👀 PR includes

🐛 Bugfix

- [x] Solve the issue and close #0

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| ❌  error  | ✅ scroll |

### CODE REVIEW

File: packages/s2-core/src/facet/base-facet.ts [hunk 1]
```diff
@@ -928,12 +928,9 @@ export abstract class BaseFacet {
           allCells,
           (cell) => cell.get('name') === `${i}-${j}`,
         );
+        updateMergedCells(this.spreadsheet);
```
Comment (xingwanying) [inline packages/s2-core/src/facet/base-facet.ts:931]: 等等……手误恢复下……

**Review by lijinke666 — DISMISSED** [review]

**Review by lijinke666 — APPROVED** [review]

**Pull request merged**